wxmedit.txt

```diff
@@ -0,0 +1,59 @@
+[Section]
+Name = wxMEdit
+Version = 3.2
+Description = A Cross-platform Text/Hex Editor
```

Comment:
Ok since this is a dev tool, description could be improved:
```suggestion
Description = A cross-platform text/hex editor written in C++ & wxWidgets, supporting various encodings even in hexadecimal mode.
```

Reply:
Agreed. And then the descriptions in each language will also need to be updated or removed. I can only understand & update the descriptions in Simplified and Traditional Chinese 😂.

Reply:
Yep, better remove them to offload this work to our translators. Feel free to translate to the languages you know. 🙂

Reply:
> Yep, better remove them to offload this work to our translators. Feel free to translate to the languages you know. 🙂

OK. Updated.